openshift/templates/nodejs.json

```diff
@@ -189,6 +189,26 @@
       "description": "A secret string used to configure the GitHub webhook",
       "generate": "expression",
       "from": "[a-zA-Z0-9]{40}"
+    },
+    {
+      "name": "DATABASE_SERVICE_NAME",
+      "description": "Database service name"
+    },
+    {
+      "name": "MONGODB_USER",
+      "description": "Username for MongoDB user that will be used for accessing the database"
+    },
+    {
+      "name": "MONGODB_PASSWORD",
+      "description": "Password for the MongoDB user"
+    },
+    {
+      "name": "MONGODB_DATABASE",
+      "description": "Database name"
+    },
+    {
+      "name": "MONGODB_ADMIN_PASSWORD",
+      "description": "Password for the database admin user"
```

Comment:
Right now it depends on DATABASE_SERVICE_NAME being non-empty (slightly different than being undefined), and ${DATABASE_SERVICE_NAME}_SERVICE_HOST/PORT, plus database name, user and password, being set to a proper value (otherwise you get a connection error).